src/ring_theory/polynomial/basic.lean

```diff
@@ -651,7 +651,70 @@ is_noetherian_submodule_left.1 (is_noetherian_of_fg_of_noetherian _
 
 end ideal
 
+section prime
+variables (σ) {r : R}
+
+namespace polynomial
+lemma prime_C_iff : prime r ↔ prime (polynomial.C r) :=
```

Comment:
Please swap the sides of the `↔` (that's also what the name suggests). Should this be a simp-lemma?

Reply:
The reason the order is like so is because I only [need the forward direction](https://github.com/leanprover-community/mathlib/pull/12866/files#diff-c87837aaf71182044c933dd9227b8f7c23bc542f6b5f38260ffbe50fba94fc15R1166) and originally only proved that direction, but later think it would be nice to have the iff statement.

I am not sure if it should be a simp lemma; that wouldn't help the various proofs below, as the main content there is to show that various maps between mv_polynomial rings preserves `C`. But I can imagine such simp lemmas may be useful somewhere. Let me first switch the order.